client/lib/url/url-type.ts

```diff
@@ -52,8 +53,11 @@ export function determineUrlType( url: URLString | URL ): URL_TYPE {
 
 	try {
 		// If we can parse the URL without a base, it's an absolute URL.
+		// The polyfill works differently, however, so we need to take that into account.
```

Comment:
Add a link to the issue in the polyfill repo?

Reply:
Good idea! 👍